pandas/core/generic.py

```diff
@@ -453,22 +453,29 @@ def _get_axis_resolvers(self, axis):
         d[axis] = dindex
         return d
 
-    def _get_index_resolvers(self):
-        d = {}
+    def _get_index_resolvers(self) -> Dict[str, ABCSeries]:
```

Comment:
as a followup I'd like to clean up all of these functions; i think we can move them entirely to pandas/computation/ (including the axis resolvers which is just plain wrong now the way its implemented), can you create an issue for this

Reply:
#30683

Reply:
and replace ABCSeries in type annotations. ABCSeries resolves to any.

Reply:
Replace with what? Also, let's continue this in #30683

Reply:
I _think_ return type is Dict[str, Union["Series", MultiIndex]]

Reply:
> Also, let's continue this in #30683

I don't think typing discussion relevant there

Reply:
Okay, I will see if that works. 

The issue is meant for cleaning up these functions. So, why not fix the typing hints as well.